module/applications/actor/base-sheet.mjs

```diff
@@ -686,18 +684,23 @@ export default class ActorSheet5e extends ActorSheet {
   /* -------------------------------------------- */
 
   /**
-   * Handle input changes to numeric form fields, allowing them to accept delta-typed inputs
+   * Handle input changes to numeric form fields, allowing them to accept delta-typed inputs, uses REGEX to determine if input is valid or not.
```

Comment:
Probably not necessary to describe the regex implementation detail here.

Reply:
Fair